scripts/scil_compute_ssst_frf.py

```diff
@@ -65,10 +65,12 @@ def _build_arg_parser():
              'fiber voxels in the automatic estimation. [%(default)s]')
 
     p.add_argument(
-        '--roi_radius', default=10, type=int,
-        help='If supplied, use this radius to select single fibers from the '
-             'tensor to estimate the FRF. The roi will be a cube spanning '
-             'from the middle of the volume in each direction. [%(default)s]')
+        '--roi_radii', default=[10], nargs='+', type=int,
+        help='If supplied, use those radii to select a cuboid roi '
```

Comment:
Maybe this is a bit verbose, maybe keep the description about a cube only? Or move some of it to the description? @karanphil can you help design something a bit shorter for Antoine?